Dockerfile

```diff
@@ -1,27 +1,24 @@
-FROM centos:7 AS base
+FROM centos:8 AS base
 # ================ Builder stage =============================================
-# we base our image on a vanilla Centos 7 image.
+# we base our image on a vanilla Centos 8 image.
 
-ENV ACS_PREFIX=/alma ACS_VERSION="2020.8"
+ENV ACS_PREFIX=/alma ACS_TAG="2020AUG" ACS_VERSION="2020.8"
 
-ENV ACS_ROOT=$ACS_PREFIX/acs
+ENV ACS_ROOT="${ACS_PREFIX}/ACS-${ACS_TAG}"
 
 ENV JAVA_HOME="/usr/java/default"
 
 # install deltarpm prior to installing everything else
 # it might save some time during downloading and installing the
 # dependencies below, but it is not urgently needed for ACS to work
 # c.f. https://www.cyberciti.biz/faq/delta-rpms-disabled-because-applydeltarpm-not-installed/
```

Comment:
This comment belongs to the removed line below .. so we should also remove it ... less comment .. less stuff to read .. yay \o/

Reply:
I could quickly do it if you like?...